extensions/eventtap/init.lua

```diff
@@ -177,29 +177,41 @@ function module.rightClick(point, delay)
     module.event.newMouseEvent(module.event.types["rightMouseUp"], point):post()
 end
 
---- hs.eventtap.middleClick(point[, delay])
```

Comment:
I was referring mainly about the function, but documentation should still exist for it. I see your point in aligning with apple's naming conventions, maybe you can refer to that in the middleClick documentation.